mlrun/frameworks/_common/plots.py

```diff
@@ -4,9 +4,22 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
-import seaborn as sns
-from scikitplot.metrics import plot_calibration_curve
-from scipy import interp
+
+try:
+    import seaborn as sns
+except ModuleNotFoundError:
+    pass
+
+try:
+    from scikitplot.metrics import plot_calibration_curve
+except ModuleNotFoundError:
+    pass
+
+try:
+    from scipy import interp
+except ModuleNotFoundError:
+    pass
```

Comment:
Why it's needed to wrap these with try except ?

Reply:
They are failing due to tests, this whole function will be re-written in the next PR